app/models/re_request_submission.rb

```diff
@@ -11,17 +11,4 @@ def is_asset_applicable_to_type?(request_type, asset)
   end
   private :is_asset_applicable_to_type?
 
-  def all_samples_have_accession_numbers?
```

Comment:
Because Multiplexed Libraries can be now registered via manifest, these assumptions are no longer valid.